serverlessworkflow/modules/ROOT/pages/cloud/operator/developing-workflows.adoc

```diff
@@ -0,0 +1,245 @@
+----
+{
+  "id": "984b5c6c-36ef-48ba-aa11-89fa54d25e98",
+  "workflowdata": {
+    "name": "Jane Doe",
+    "greeting": "Hello from JSON Workflow, "
+  }
+}
+----
+
+You can even make changes to your `KogitoServerlessWorkflow` YAML file and see the results using the Swagger UI.
+
+[[troubleshooting]]
+== Troubleshooting the Workflow Service
+
+Since during development you are iterating over the deployed workflow service, it's likely that you will need to troubleshoot the application if something goes wrong.
+
+To ensure the workflow is running in a healthy state, the operator deploy the application pod with health checks probes.
+So if your changes impact the application somehow, the pod will stop responding.
+
+[[basic-troubleshooting]]
+=== Basic Troubleshooting
+
+1. Analyze the workflow status with:
++
+.Get the workflow status conditions
+[source,shell,subs="attributes+"]
+----
+kubectl get workflow <name> -o jsonpath={.status.conditions} | jq .
+----
++
+It can give you a clue about what might be happening. See xref:cloud/operator/workflow-status-conditions.adoc[] for more information.
++
+2. Fetch the logs and look for `ERROR` messages:
++
+.Watch the application logs
+[source,shell,subs="attributes+"]
+----
+kubectl logs deployment/<workflow-name> -f
+----
++
+If you are looking for opening an issue or ask in {product_name} communication channels, this logging information is always useful for the person who will try to help you.
+
+=== Possible Failure Scenarios
+
+==== Feature Not Yet Supported 
+The {operator_name} is under active development. Sometimes a feature might not be available yet. Please see xref:cloud/operator/known-issues.adoc[] for a compreehensive list of available features.
+
+If you identify you're refering to a feature not yet available, please file a new issue so we can prioritize it for you or ask in {product_name} communication channels.
+
+==== Wrong Application Configuration
+A wrong configuration, or lack of one might impact your workflow to run correctly. 
+The operator deploys a `ConfigMap` that holds the application properties for the workflow. 
+
+.Get the properties ConfigMap
+[source,shell,subs="attributes+"]
+----
+kubectl get cm <workflow-name>-props
+----
+
+The `ConfigMap` name pattern is the workflow name followed by `-props`.
+
+Make sure that the configuration is correct and you're not lacking any required properties for a given feature to work. 
+If so, you can make your changes to the configuration as you normally would to any `ConfigMap`. 
+
+The operator ensures that these properties are applied to the application.
+
+See xref:cloud/operator/configure-workflows.adoc[] for more information.
+
+==== Wrong Workflow Definition
+The {operator_name} validates the workflow definition at the moment you create or edit the YAML file, avoiding to persist a workflow in an invalid state. 
+Although, the operator is under active development, so errors during the validation might occur. 
```

Comment:
maybe link the admission webhook issue here?

Reply:
It's in the "Known issues", linked above.

Reply:
I saw that later =/